README.md

```diff
@@ -67,6 +67,47 @@ The following may also be added to your inventory.
 collectd1.somedc.prod         ansible_ssh_host=10.0.1.111   private_ip=172.16.1.111
 ```
 
+### Plugin configuration
+
+#### [Disk](https://collectd.org/wiki/index.php/Plugin:Disk)
+
+
+Default: xvde
+
+```
+check_disk: xvde
+```
+
+#### [DF](https://collectd.org/wiki/index.php/Plugin:DF)
+
+All options are optional.
+
+For Collectd 5.4, define `fs_type`. Example `fs_type: ext4`. Note: Do not set `fs_type` if you're using Collectd 5.5.
```

Comment:
Why having 2 different variables for the same thing?
Are the options for collectd 5.5 not supported in collectd 5.4?